### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by a2-sharma**
Updated - main structure now in place.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,146 @@
...
+* Contract
+  * Analogy: Summer romance
+  * You are hired to deliver a specific task within a certain amount of time.  There is not necessarily an expectation that you are part of regular team meetings and you may be working on items which are outside the regular company workflow.
+  * Jobs may be advertised, but will often accrue through contacts and networks.
+  * Contracts likely in the 3-6 month timeframe.
+  * You are highly unlikely to get any benefits from the company or stock options.
+  * It's very important to make the expectations clear - most contracts will not have this 100% clear at the start, and this is important to clarify - what has to be delivered by when, and when payments are made.
+* Freelance
+  * Upwork etc - not covered here.
+
```
Comment (Abuton) [inline README.md:66]: Should i add more options here or leave as is

**Review by Abuton — CHANGES_REQUESTED** [review]
The format change a bit.  All addition are well thought

I have made some changes and added some contents as well

File: README.md (hunk 1 above)
Comment (a2-sharma) [reply README.md:66]: If you have more options, please do add.

**Comment by a2-sharma** [comment]
@Abuton back to you!

**Pull request merged**